src/js/utils/polyfills.js

```diff
@@ -1,3 +1,7 @@
+if (!global._babelPolyfill) {
```

Comment:
Should be evaluated at runtime... but that's mainly why I want to test it in IE.. just to make sure.